helpers/craigslist_scraper.rb

```diff
@@ -0,0 +1,77 @@
+require 'mechanize'
+require_relative 'posting'
+
+class CraigslistScraper
+  attr_reader :search_page, :scraper, :results_page, :postings
+
+  def initialize(min_price, max_price, keywords)
+    @scraper = Mechanize.new
+    @scraper.history_added = Proc.new { sleep 0.5 }
+    @search_page = scraper.get('http://sfbay.craigslist.org/search/apa')
```

Comment:
Does this work when you set scraper as an instance variable on line 8 and then you are using it as a local variable on line 10.